src/commonMain/kotlin/kotools/types/number/NonZeroInt.kt

```diff
@@ -64,6 +65,11 @@ public sealed interface NonZeroInt : AnyInt {
     override fun toString(): String
 }
 
+/** Returns the negative of this integer. */
+@ExperimentalNumberApi
+@SinceKotoolsTypes("4.2")
+public operator fun NonZeroInt.unaryMinus(): Int = -this.toInt()
```

Comment:
- Like indicated in the issue #54, this operation should return a `NonZeroInt`.
- You can remove the `this` keyword in this implementation.